jcl/src/java.base/share/classes/java/lang/J9VMInternals.java

```diff
@@ -150,7 +151,9 @@ private static void completeInitialization() {
 						} catch (Throwable e) {
 							/* empty block */
 						}
+						/*[IF JAVA_SPEC_VERSION < 26]*/
 						break;
+						/*[ENDIF] JAVA_SPEC_VERSION < 26 */
```

Comment:
I don't think `break;` is appropriate for any version; there may be more than one thread with the name "Common-Cleaner".